### ISSUE TITLE
var in for loop

### ISSUE THREAD

**Issue by gknops** [douglascrockford/JSLint#123]
Hi, I believe this is perfectly fine (actually good form to limit variable scope) and should be allowed:

``` javascript
for(var i=0;i<10;i++)
{
    // do something with i
}
```

but JSLint doesn't tolerate the `var` in there.
